server/src/main/java/io/spine/server/rejection/model/RejectionSubscriberMethod.java

```diff
@@ -57,20 +59,29 @@
      * Invokes the wrapped handler method to handle {@code rejectionMessage},
      * {@code commandMessage} with the passed {@code context} of the {@code Command}.
      *
-     * @param target
-     *        the target object on which call the method
-     * @param rejectionMessage
-     *        the rejection message to handle
-     * @param context
-     *        the context of the rejection
+     * @param target           the target object on which call the method
+     * @param rejectionMessage the rejection message to handle
+     * @param context          the context of the rejection
      */
     @CanIgnoreReturnValue
     @Override
-    public Object invoke(Object target, Message rejectionMessage, RejectionContext context) {
-        ensureExternalMatch(this, context.getExternal());
+    public
+    MethodResult<Empty> invoke(Object target, Message rejectionMessage, RejectionContext context) {
+        ensureExternalMatch(context.getExternal());
+        doInvoke(target, rejectionMessage, context);
+        return MethodResult.empty();
+    }
 
-        Object result = doInvoke(target, rejectionMessage, context);
-        return result;
+    /**
+     * @implNote This method is never called because
```

Comment:
Seems like this tag fails our CI builds.

Reply:
I initially removed the tags in the source, but now enabled the tag options for `javadoc` in `build.gradle`. So, it should work now. It's better with new tags.